job-server/src/main/resources/application.conf

```diff
@@ -131,3 +131,19 @@ spray.can.server {
   # Increase this in order to upload bigger job jars
   parsing.max-content-length = 30m
 }
+
+shiro {
+    # activate authentication (and authorization)
+    # authentication = on
+    # absolute path to shiro config file, including file name
+    path.ini = "/some/path/shiro.ini"
+    # note that ssl-encryption should also be on when authentication is on
```

Comment:
No, not related to LDAP. I modified the comment as follows:
# note that ssl-encryption should also be on when authentication is on to avoid that passwords
# can be sniffed out